modules/web/src/app/node-data/basic/provider/kubevirt/template.html

```diff
@@ -114,8 +114,8 @@
                [options]="subnets"
                [formControlName]="Controls.Subnet"
                [label]="subnetLabel"
-               hint="Optional: Use specific subnet for the machines. VPC needs to be selected first to use this option."
-               [required]="false"
+               hint="Use specific subnet for the machines. VPC needs to be selected first to use this option."
```

Comment:
This description looks wrong. It says VPC is required for this option so should we disable this when VPC is not selected?

Reply:
Right now, if no VPC is selected, there are no subnets, so the field is always empty and not required. The label says "No subnets available or VPC not selected," which I thought would make it clear to the user that they need to select a VPC to see subnets. So I figured maybe there's no need to disable the field, since it has no subnets anyway.

Reply:
i wanted to remove the `Optional` cause if VPC is selected then it's not optional